CHANGELOG.md

```diff
@@ -26,14 +26,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
 
-- Added: 3 videos to the logic the database for a diagonal bomb jump in Ghavoran, a single-wall jump in Cataris, and a diffusion abuse trick in Artaria.
-
+- Added: 3 videos to the logic database for a diagonal bomb jump in Ghavoran, a single-wall jump in Cataris, and a diffusion abuse trick in Artaria.
+- Added: 2 videos to the logic database for cross bomb movement in Burenia.
+- Added: Cross Bomb Launch (CBL) is now its own trick in the database. 
 ##### Artaria
 
 - Changed: EMMI Zone Spinner: The connection to the pickup that is available before flipping the spinner now also requires door lock rando and Highly Dangerous Logic to be enabled.
 
 ##### Burenia
 
+- Added: The missile tank in Burenia Main Hub Tower Top can be gotten with morph, cross bombs, and movement (intermediate).
```

Comment:
I just approved, but i forgot about this. Should be Cross Bomb Launch instead of Movement.

Reply:
Changed.